libafl/src/events/centralized.rs

```diff
@@ -324,7 +324,7 @@ where
 impl<E, EM, SP, Z> EventManager<E, Z> for CentralizedEventManager<EM, SP>
 where
     EM: EventStatsCollector + EventManager<E, Z>,
-    EM::State: HasClientPerfMonitor + HasExecutions + HasMetadata + HasLastReportTime,
+    EM::State: HasClientPerfMonitor + HasExecutions + HasMetadata + HasLastReportTime + HaF,
```

Comment:
I cannot find `HaF` in the source, likely a typo?

Reply:
No, It is my mistake.  [29b1122](https://github.com/AFLplusplus/LibAFL/pull/1351/commits/29b1122fc12a9b3f3ba56c648cc0d93cf49da185) is not yet good to merge into the main branch. Some functions of stats_stage still need to be reconstructed.